packages/generator-widget/generators/app/templates/packages/package_web.json.ejs

```diff
@@ -40,11 +40,13 @@
     "classnames": "^2.2.6"
   },
   "resolutions": {
-    "react": "18.2.0",
+    "react": "18.2.0",<% if (isLanguageTS) { %>
+    "@types/react": "18.2.79",<% } %>
     "react-native": "0.72.7"
   },
   "overrides": {
-    "react": "18.2.0",
+    "react": "18.2.0",<% if (isLanguageTS) { %>
+    "@types/react": "18.2.79",<% } %>
```

Comment:
I am wondering why this particular version was chosen and whether the React override should be updated as well.

Reply:
Since it appears we can use ranges in [the overrides](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides), we could follow "^18.2.0" for both `react` and `@types/react`